sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Customizations/ScriptActivity.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Azure.Management.DataFactory.Models
+{
+    public partial class ScriptActivity : ExecutionActivity
+    {
+        /// <summary>
+        /// Initializes a new instance of the ScriptActivity class.
+        /// </summary>
+        /// <param name="name">Activity name.</param>
+        /// <param name="additionalProperties">Unmatched properties from the
+        /// message are deserialized this collection</param>
+        /// <param name="description">Activity description.</param>
+        /// <param name="dependsOn">Activity depends on condition.</param>
+        /// <param name="userProperties">Activity user properties.</param>
+        /// <param name="linkedServiceName">Linked service reference.</param>
+        /// <param name="policy">Activity policy.</param>
+        /// <param name="scripts">Array of script blocks. Type: array.</param>
+        /// <param name="logSettings">Log settings of script activity.</param>
+        public ScriptActivity(string name, IDictionary<string, object> additionalProperties = default(IDictionary<string, object>), string description = default(string), IList<ActivityDependency> dependsOn = default(IList<ActivityDependency>), IList<UserProperty> userProperties = default(IList<UserProperty>), LinkedServiceReference linkedServiceName = default(LinkedServiceReference), ActivityPolicy policy = default(ActivityPolicy), IList<ScriptActivityScriptBlock> scripts = default(IList<ScriptActivityScriptBlock>), ScriptActivityTypePropertiesLogSettings logSettings = default(ScriptActivityTypePropertiesLogSettings))
```

Comment:
Same as above, we should change the parameters to be mandatory that are before the newly inserted parameter and also the parameter which has the same location as the new parameter. In thie ctor, the parameters from `additionalProperties` to `scripts` should not be optional parameter anymore.

Reply:
fix